### ISSUE TITLE
Does fscrawler recognize the file language?

### ISSUE THREAD

**Issue by PakanAngel** [dadoonet/fscrawler#469]
I 'v configured fscrawler for crawling some folders. Some of my repos. include files in Arabic and Persian. Does Fscrawler have any property to set for these languages or do I have  to set it by elasticsearch properties?

**Comment by dadoonet** [comment]
Yes. You probably need to set a text analyzer that suits your need.

Have a look at Creating your own mapping (analyzers) in the Readme

**Comment by bobjiang1988** [comment]
The document said that `If you want to define your own index settings and mapping to set analyzers for example, you can update the needed component template before starting the FSCrawler.`

But, `_component_template/fscrawler_mapping_content` will be override by after FSCrawler started.

**Comment by dadoonet** [comment]
@bobjiang1988 I think it's a bug actually. I mean a misconception. Could you open a new issue?

**Comment by dadoonet** [comment]
Coming back to the discusion, this is actually documented (see https://fscrawler.readthedocs.io/en/latest/admin/fs/elasticsearch.html#mappings):

> You can stop FSCrawler creating/updating the index templates for you by setting `push_templates` to `false`:
> 
> ```yml
> name: "test"
> elasticsearch:
>   push_templates: false
> ```
